sat/cli/bootsys/bos.py

```diff
@@ -960,7 +981,7 @@ def do_bos_reboots(args: Namespace):
 
     Returns: None
     """
-    if not args.disruptive:
+    if not (args.disruptive or args.staged_session):
```

Comment:
FYI, @jesseviola-hpe. It looks like this means you won't need to pass `--disruptive` if doing a staged session. Eli can confirm that I'm interpreting this correctly.

Reply:
Correct, I felt the prompt was pretty misleading in the staged case, since it might lead you to believe you were about to do an actual reboot/shutdown.